apps/server/src/modules/system/service/system.service.ts

```diff
@@ -8,7 +8,7 @@ import { SystemDto } from '@src/modules/system/service/dto/system.dto';
 export class SystemService {
 	constructor(private readonly systemRepo: SystemRepo) {}
 
-	async find(type: string | undefined): Promise<SystemDto[]> {
+	async find(type: SystemType | undefined): Promise<SystemDto[]> {
```

Comment:
findAll or something like that would be a better name, as find, same like save, delete, update I would keep for single record